kuma/javascript/src/payments/pages/terms.jsx

```diff
@@ -58,27 +51,34 @@ const TermsPage = ({ data: { contributionSupportEmail } }: Props) => (
                     >
                         Privacy Policy
                     </a>
```

Comment:
Needs punctuation and space after "Privacy Policy":
<img width="493" alt="Screen Shot 2020-05-29 at 1 41 19 PM" src="https://user-images.githubusercontent.com/650/83289043-68a33b00-a1b2-11ea-946b-b864f7be97bc.png">

Reply:
fine. do I have to do anything to approve it?

Reply:
No, just so you know for future's sake-- usually you would make the change and update the diff and re-request a review from me. 

I went ahead and made the change since I was already in your branch and today is my last day, so I didn't want to leave this PR hanging. I'll merge when the build passes. Thanks for making these changes!